job-runner/runner/factory.go

```diff
@@ -11,7 +11,7 @@ import (
 // A Factory takes a proto.Job, creates a corresponding job.Job interface for
 // it, and passes both to NewRunner to make a Runner.
 type Factory interface {
-	Make(job proto.Job, requestId string, prevTryNo uint, sequenceRetry uint) (Runner, error)
+	Make(job proto.Job, requestId string, prevTryNo uint, triesToSkip uint, sequenceRetry uint) (Runner, error)
```

Comment:
The var name `triesToSkip` is a little misleading. We never skip tries. In this case, we've already done those tries. Not sure what a better name is, but we need to think of one.